### DISCUSSION TITLE
OpenDTU Breakout Board

### DISCUSSION THREAD

**Discussion by dokuhn** [tbnobody/OpenDTU#197] [Show and tell]
Zuerst muss ich sagen, dass ich OpenDTU ein klasse Projekt finde. Aus diesem Grund habe ich mal ein [Breakout Board](https://github.com/dokuhn/openDTU-BreakoutBoard) in KiCAD gebaut. Gerne mitdiskutieren, teilen und Ideen für eine Hardwarebasis für dieses Projekt.

**Comment by kpwg** [comment]
Hallo Dominik,

auch ich habe mir bereits Gedanken zur Platine gemacht. Folgende Punkte fallen mir spontan dazu ein:

-   Etablierung eines "Standard-Pinnings", d.h. Übernahme der in openDTU standardmäßigen Verdrahtung ins Layout
-   Welche ESP32-BoardVariante ist die Gängigste? Es gibt 30pin, 38pin, D1-Mini-ähnlich, ...
-   Konstruktion der Platine zur sicheren Montage in einem käuflichen ABS-Gehäuse o.ä. (nicht jeder kann drucken)
-   sollen die beiden HF-Teile wirklich so dicht beieinander sein? Gibt es da bereits Erfahrungen? Ich wüsste spontan nicht, wie man die beiden Antennen des ESP32 und des NRF soweit wie möglich auseinander bekommt, ohne das Layout zu "vermurksen". Es wird sicher ein Kompromiss :)
-   Stromversorgung über den ESP ist sicher ok, aber zusätzlich auf der Platine sollte noch Platz für einen stepdown Schaltregler sein. Ich denke da z.B. an ein Modul mit MP1584EN, was sehr verbreitet ist. Davor dann eine Verpolschutzdiode und Platz für eine Supressor-Diode als Überspannungsschutz. Ein kleiner Elko nach dem Regler ist denke ich selbstverständlich. Elektrisch zugänglich von außen würde ich das Ganze über eine 2pol. Schraubklemme machen.

**Reply by dokuhn** [reply]
Ich hab jetzt mal die erste Variante soweit finalisiert, sodass sie bei dem Distributor des Vertrauens bestellt werden kann. Deine Anregungen werde ich gerne in einer zweiten Variante (neuer Branch) mal zur Entwicklung aufnehmen. Dachte da auch auf Dauer und der Einfachheit halber an ein fertiges Gehäuse. Bspw. ganz schick mit halbdurchsichtigen Deckel wie die der Firma [Spelsberg](https://www.spelsberg.de/industrieleergehaeuse/glatt-mit-befestigungsnocken/20100401/).

Aber auch die Dinge wie ordentliche separate Stromversorgung mittels Schaltregler ist sinnvoll. Ich werde in den nächsten Wochen mal dafür ein Branch anlegen, dann kann man das ein oder andere einfließen lasse.
